store/map/index.js

```diff
@@ -210,13 +192,9 @@ export const getters = {
       .filter(identity)
   },
 
-  activeTimestamp(state, { activeSpatialData }) {
-    if (
-      activeSpatialData &&
-      activeSpatialData.length &&
-      activeSpatialData[0] !== ''
-    ) {
-      const str = activeSpatialData[0]
+  activeTimestamp(state, { activeRasterData }) {
+    if (get('[0]', activeRasterData)) {
```

Comment:
`_.head(activeRasterData)`?

Reply:
Yes, done this in Feat handling multiple mapboxlayers per dataset